### PR TITLE
footer updates

### PR DESCRIPTION
**Pull request by kavitappatil**

### CODE REVIEW

File: langjo-frontend/src/Components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                </small>
+              </div>
+            </li>
+          </ul>
+        </nav>
+      </div>
+      <div className="d-flex flex-column flex-md-row p-4 gap-5 py-md-5 align-items-center justify-content-center">
+        &nbsp; &nbsp;
+      </div>
+      <div
```
Comment (JanefrancessC) [inline langjo-frontend/src/Components/Footer.js:28]: Is it possible to center this div?  Then the logo colors should be reflected

File: langjo-frontend/src/Components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        className="position-static d-flex flex-column flex-lg-row align-items-stretch
+        justify-content-start p-3 rounded-3 shadow-lg"
+      >
+        <label className="d-flex flex-column">
+          <div class="col">
+            <div className="logo">LangJo</div>
```
Comment (JanefrancessC) [inline langjo-frontend/src/Components/Footer.js:34]: Can you add the logo color?

File: langjo-frontend/src/Components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                </a>
+                &nbsp; &nbsp;&nbsp; &nbsp;
+                <a href="#">
+                  <i className="bi bi-instagram icon-white icon-large"></i>
+                </a>
+              </div>
+              &nbsp; &nbsp;
+            </div>
+            <div className="py-3">
+              <div className="cookie">Private Policy | Cookie Notice</div>{" "}
```
Comment (JanefrancessC) [inline langjo-frontend/src/Components/Footer.js]: It should be "Privacy Policy | Cookie Notice".
Also, the colors should be as in the Figma design

**Review by JanefrancessC — COMMENTED** [review]
You've done a neat and pretty job.
The responsive design isn't very good yet.

File: langjo-frontend/src/Components/Footer.js (hunk 1 above)
Comment (kavitappatil) [reply langjo-frontend/src/Components/Footer.js:28]: We need a proper logo for the site.

File: langjo-frontend/src/Components/Footer.js (hunk 2 above)
Comment (kavitappatil) [reply langjo-frontend/src/Components/Footer.js:34]: We need a proper logo for the site.

File: langjo-frontend/src/Components/Footer.js (hunk 3 above)
Comment (kavitappatil) [reply langjo-frontend/src/Components/Footer.js]: yes, fixed the "Privacy Policy | Cookie Notice" and colour , thanks

**Comment by PakizeBozkurt** [comment]
Updated new ticket

**Pull request closed without merging**